src/main/java/me/totalfreedom/totalfreedommod/command/Command_smite.java

```diff
@@ -80,9 +82,11 @@ public static void smite(Player player, String reason)
         // Kill
         player.setHealth(0.0);
 
+        player.sendMessage(ChatColor.YELLOW + "You've been smitten by: " + sender.getName());
         if (reason != null)
         {
-            player.sendMessage(ChatColor.RED + "You've been smitten. Reason: " + ChatColor.YELLOW + reason);
+            player.sendMessage(ChatColor.YELLOW + sender.getName() + " smited you for, " + reason);
```

Comment:
the sender is already shown in the chat to all players.  don't  need to edit this.